src/main/java/org/cactoos/func/RepeatedFunc.java

```diff
@@ -75,19 +75,17 @@ public RepeatedFunc(final Func<X, Y> fnc, final int max) {
 
     @Override
     public Y apply(final X input) throws Exception {
+        if (this.times <= 0) {
+            throw new IllegalArgumentException(
+                "The number of repetitions must be at least 1"
+            );
+        }
         Y result = null;
         for (int idx = 0; idx < this.times; ++idx) {
             result = this.func.apply(input);
         }
-        if (result == null) {
-            throw new IllegalArgumentException(
-                new FormattedText(
-                    "Repeat counter is equal or less than zero: %d",
-                    this.times
-                ).asString()
-            );
-        }
-        return result;
+        final Y rslt = result;
+        return new Scalar.NoNulls<Y>(()->rslt).value();
```

Comment:
@ixmanuel why do we need that? Why can't that function return `null`? I think this is something our users will complain about. This functionality is not what the class is supposed to do.